### ISSUE TITLE
AZ State Spot Check Issues

### ISSUE THREAD

**Issue by JhaelleP** [openstates/issues#304]
Short Description: When checking Arizona’s bill data, there were some sponsors listed twice, and the action's dates for one bill is inaccurate

Include these 3 sections as appropriate (or note none)

Missing or Incorrect legislators:
N/A

Data Issues:
On bills HCR2016 and HR2001 the bill sponsor is listed twice
On the bill SB 1155 OpenStates and Enview have the same actions, but they have different dates than the legislative website

Additional Data:
N/A

**Comment by showerst** [comment]
https://openstates.org/az/bills/55th-1st-regular/SB1155/

> On the bill SB 1155 OpenStates and Enview have the same actions, but they have different dates than the legislative website

Can you provide examples? These seem to match up to me.

> On bills HCR2016 and HR2001 the bill sponsor is listed twice

https://openstates.org/az/bills/55th-1st-regular/HCR2016/
https://openstates.org/az/bills/55th-1st-regular/HR2001/

Which one? They look OK to me.

**Comment by JhaelleP** [comment]
When I check again, I do not see any discrepancies between the two either. I can only assume that things changed between the time I first discovered this issue and when I reported it now. I'll be sure to check on the accuracy again right before I post a bug. 
When reporting about the bill sponsors I was referring to Enview, I did not realize reporting these issues here was exclusive to OpenStates. Thank you for checking up on my reported issue.
